mvn-extension/src/main/java/org/arquillian/smart/testing/mvn/ext/dependencies/DependencyResolver.java

```diff
@@ -68,6 +68,7 @@ private void addStrategies(Model model) {
                 + "for the list of available strategies.");
         }
         configuration.setStrategies(strategies);
+        configuration.setStrategiesConfiguration(configuration.strategyConfigurations(strategies));
```

Comment:
This seems to be a bit hacky - calling on an instance of the configuration object a method and assign the result to the very same instance... it can be assigned directly inside of the method call...